packages/tpetra/core/src/Tpetra_Details_WrappedDualView.hpp

```diff
@@ -29,8 +29,8 @@
   if (envVarSet && (std::strcmp(envVarSet,"1") == 0)) \
     std::cout << (fn) << " called from " << callerstr \
               << " at " << filestr << ":"<<linnum \
-              << " host cnt " << dualView.h_view.use_count()  \
-              << " device cnt " << dualView.d_view.use_count()  \
+              << " host cnt " << dualView.view_host().use_count()  \
```

Comment:
I’m not sure if it’s an issue, but here, behavior changes, right? Because the getter is making a copy, so we’ll now report the previous use count incremented by one.

Reply:
Yes, in general all use counts will be larger but from what I understand only the difference between host and device is for relevance here.